en/deploy-tidb-cluster-across-multiple-kubernetes.md

```diff
@@ -0,0 +1,670 @@
+    replicas: 1
+    service:
+      type: ClusterIP
+    tlsClient:
+      enabled: true
+    config:
+      security:
+       cert-allowed-cn:
+         - TiDB
+EOF
+```
+
+## Exit and reclaim clusters that already join a cross-Kubernetes cluster
+
+When you need to make a cluster exit from the joined TiDB cluster deployed across Kubernetes and reclaim resources, you can perform the operation by scaling in the cluster. In this scenario, the following requirements of scaling-in need to be met.
+
+- After scaling in the cluster, the number of TiKV replicas in the cluster should be greater than the number of `max-replicas` set in PD. By default, the number of TiKV replicas needs to be greater than three.
+
+Take the cluster #2 created in [the last section](#deploy-a-new-cluster-to-join-the-initial-cluster) as an example. First, set the number of replicas of PD, TiKV, and TiDB to `0`. If you enable other components such as TiFlash, TiCDC, and Pump, set the number of these replicas to `0`:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+kubectl patch tc cluster2 --type merge -p '{"spec":{"pd":{"replicas":0},"tikv":{"replicas":0},"tidb":{"replicas":0}}}'
+```
+
+Wait for the status of cluster #2 to become `Ready`, and scale in related components to `0` replica:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+kubectl get pods -l app.kubernetes.io/instance=cluster2 -n pingcap
+```
+
+The Pod list shows `No resources found`. At this time, Pods have all been scaled in, and cluster #2 exits the cluster. Check the cluster status of cluster #2:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+kubectl get tc cluster2
+```
+
+The result shows that cluster #2 is in the `Ready` status. At this time, you can delete the object and reclaim related resources.
+
+{{< copyable "shell-regular" >}}
+
+```bash
+kubectl delete tc cluster2
+```
+
+Through the above steps, you can complete exit and resources reclaim of the joined clusters.
+
+## Enable the existing data cluster across multiple Kubernetes cluster feature as the initial TiDB cluster
+
+> **Warning:**
+>
+> Currently, this is an experimental feature and might cause data loss. Please use it carefully.
+
+1. Update `.spec.clusterDomain` configuration:
+
+    Configure the following parameters according to the `clusterDomain` in your Kubernetes cluster information:
+
+    > **Warning:**
+    >
+    > Currently, you need to configure `clusterDomain` with correct information. After modifying the configuration, you can not modify it again.
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    kubectl patch tidbcluster cluster1 --type merge -p '{"spec":{"clusterDomain":"cluster1.com"}}'
+    ```
+
+    After completing the modification, the TiDB cluster performs the rolling update.
+
+2. Update the `PeerURL` information of PD:
+
+    After completing the rolling update, you need to use `port-forward` to expose PD's API interface, and use API interface of PD to update `PeerURL` of PD.
```

Comment:
```suggestion
    After completing the rolling update, you need to use `port-forward` to expose PD's API and use the API to update `PeerURL` of PD.
```

Reply:
API is short for application programming interface. There's no need to say `API interface`. Please update other expressions in this doc as well.